topics/perovskite/index.md

```diff
@@ -0,0 +1,9 @@
+---
+aliases: perovs
+created_by: Paramesh Chandra
+display_name: Perovskite
+short_description: An material science research branch which deals with a specialy structured material and exploring its properties. 
```

Comment:
Better, thanks 👍🏻 

A little more spelling/grammar workshopping while we're at it:

```suggestion
short_description: A material science research branch which deals with exploring the properties of a specially structured material. 
```